py/mpprint.h

```diff
@@ -39,6 +39,12 @@
 #define PF_FLAG_ADD_PERCENT       (0x100)
 #define PF_FLAG_SHOW_OCTAL_LETTER (0x200)
 
+#if MICROPY_PY_IO
+#    define MP_PRINT mp_sys_stdout_print
```

Comment:
MP_PRINTER sounds good to me, but I'd make it include the `&` in the macro definition in case it changes one day to use indirection (eg with a runtime-modifiable stdout stream).